Runtime.md

```diff
@@ -0,0 +1,91 @@
+This guide will describe how GamesRunnr runs upkeep functions and how FSPAnalyzer is used to schedule them in GameStartr projects.
+
+### Table of Contents
+1. [GamesRunnr](#gamesrunnr)
+    1. [Games](#games)
+    2. [Interval and Speed](#interval-and-speed)
+    3. [Pause and Play](#pause-and-play)
+2. [FPSAnalyzer](#fpsanalyzer)
+
+# GamesRunnr
+
+GamesRunnr is a module for running a series of callbacks on timed intervals.
+These callbacks deal with maintaining the state of the game.
+An example being a callback to redraw the canvas.
+These callbacks separately are referred to as games.
+
+## Games
+
+`upkeep` is the biggest part of GamesRunnr.
+This function deals with running all the stored games.
```

Comment:
> deals with

I like this phrase in moderation referring to large-scale functionality, but it's a little too broad & vague for describing an individual function.